src/LiveQuery/ParseLiveQueryServer.js

```diff
@@ -175,26 +176,45 @@ class ParseLiveQueryServer {
               if (!isMatched) {
                 return null;
               }
+              aclError = false;
               res = {
                 event: 'Delete',
                 sessionToken: client.sessionToken,
                 object: deletedParseObject,
                 clients: this.clients.size,
                 subscriptions: this.subscriptions.size,
                 useMasterKey: client.hasMasterKey,
-                installationId: client.installationId
+                installationId: client.installationId,
+                sendEvent: true,
               };
               return maybeRunAfterEventTrigger('afterEvent', className, res);
             })
             .then(() => {
+              if (!res.sendEvent) {
+                return;
+              }
               if (res.object && typeof res.object.toJSON === 'function') {
                 deletedParseObject = res.object.toJSON();
                 deletedParseObject.className = className;
               }
               client.pushDelete(requestId, deletedParseObject);
             })
             .catch(error => {
-              logger.error('Matching ACL error : ', error);
+              if (aclError) {
```

Comment:
Wouldn't it be better done by examining the `error` object? I am also not really sure if we should log it as an error.

Reply:
Thanks for the review @davimacedo. I wasn't sure what the ACL error would look like. Is this correct?

```
  if (
    error.code == Parse.Error.OBJECT_NOT_FOUND ||
    error.code == Parse.Error.OPERATION_FORBIDDEN
  )
```

Reply:
I actually believe that it does not throw any error but I've considered that some of your tests were failing because you've seen some error here :)

Reply:
It looks it really does not throw an error and you don't need this verification:
https://github.com/parse-community/parse-server/blob/master/src/LiveQuery/ParseLiveQueryServer.js#L531

Reply:
Ok cool, thanks for digging that up. Is it even worth having the `logger.error('Matching ACL error : ', error)` line?

Reply:
I believe that this line should never execute because of an ACL matching error. It will return false instead of throwing an error so the log should not be necessary.